elide-core/src/main/java/com/yahoo/elide/core/filter/expression/InMemoryFilterVisitor.java

```diff
@@ -5,23 +5,23 @@
  */
 package com.yahoo.elide.core.filter.expression;
 
-import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.core.RequestScope;
 
 import java.util.function.Predicate;
 
 /**
  * Visitor for in memory filterExpressions
  */
 public class InMemoryFilterVisitor implements Visitor<Predicate> {
-    private final EntityDictionary dictionary;
+    private final RequestScope requestScope;
 
-    public InMemoryFilterVisitor(EntityDictionary dictionary) {
-        this.dictionary = dictionary;
+    public InMemoryFilterVisitor(RequestScope requestScope) {
```

Comment:
Should this constructor take the Interface `RequestScope` instead of implementation one?